lib/ipmi_backend_utils.pm

```diff
@@ -303,4 +303,26 @@ sub set_serial_console_on_vh {
 
 }
 
+#Huawei arm machines require password login after "boot from local disk" is selected.
+#Before the desired grub menu is shown, you need to navigate into the boot menu and
+#select the "sles" boot item.
+sub boot_local_disk_arm_huawei {
+    my $type_speed = 20;
```

Comment:
Please use the constants defined in lib/testapi.pm, but we actually have functions/methods for this, just give it a look to lib/utils :) (And look at the suggestion below)

Reply:
Thanks.